site/app/templates/polls/PollForm.twig

```diff
@@ -0,0 +1,403 @@
+{% set url_path = poll is not null ? "/editPoll/submitEdits" : "/newPoll" %}
+{% set question_type = poll is not null ? poll.getQuestionType() : 'single-response-multiple-correct' %}
+
+<div class="content">
+    <h1>{{ poll is not null ? "Edit Poll" : "Add a New Poll" }}</h1>
+
+    <form id="new-poll-form" method="post" action="{{ base_url }}{{ url_path }}" enctype="multipart/form-data">
+        {% if poll is not null %}
+            <input type="hidden" name="poll_id" value="{{ poll.getId() }}"/>
+        {% endif %}
+        <input type="hidden" name="csrf_token" value="{{ csrf_token }}"/>
+
+        <label for="poll-name" class="option-title">
+            Poll Name:
+        </label>
+        <input type="text" name="name" id="poll-name" placeholder="Enter name here..." value="{{ poll is not null ? poll.getName() }}" />
+        <br/> <br/>
+
+        <p class="option-title">Question:</p>
+        {% include "misc/MarkdownArea.twig" with {
+            "markdown_area_id" : "poll-question",
+            "markdown_area_query" : "$('#poll-question')",
+            "markdown_area_name" : "question",
+            "markdown_area_value" : poll is not null ? poll.getQuestion()  : "",
+            "placeholder" : "Enter question here...",
+            "preview_div_id" : "poll_preview",
+            "preview_div_name" : "poll_preview",
+            "onclick" : "previewPollMarkdown.call(this)",
+            "render_buttons" : true,
+            "min_height" : "100px"
+        } only %}
+        <br/><br/>
+
+        <span class="option-title" >
```

Comment:
Code segment here looks to be identical to the code in PollType.twig - might want to put it into a separate twig file and include

Reply:
`PollType.twig` actually appears to be entirely unused, so I will delete it altogether.